libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -940,6 +942,7 @@ DNNModel *ff_dnn_load_model_tf(const char *model_filename, DNNFunctionType func_
 
     return model;
 err:
+    model->model = tf_model;
```

Comment:
`ff_dnn_free_model_tf` will not correctly free `tf_model`, if it's not assigned to `model->model`.